pymc3/variational/advi_minibatch.py

```diff
@@ -334,30 +477,22 @@ def get_transformed(v):
     )
```

Comment:
`v in ds` does not mean it is transformed, I can provide a custom expression to Deterministic function or they cannot be in local vars? This is not clear for me. Maybe just check `hasattr(var, 'transformed')`?
```python
ds = model.deterministics
 
     def get_transformed(v):
         if v in ds:
             return v.transformed
         return v
```